src/goto-analyzer/unreachable_instructions.cpp

```diff
@@ -251,12 +267,9 @@ static void json_output_function(
 {
   json_objectt entry{
     {"function", json_stringt(function)},
-    {"file name",
-     json_stringt(concat_dir_file(
-       id2string(first_location.get_working_directory()),
-       id2string(first_location.get_file())))},
-    {"first line", json_numbert(id2string(first_location.get_line()))},
-    {"last line", json_numbert(id2string(last_location.get_line()))}};
+    {"file name", json_stringt{file_name_string(first_location)}},
```

Comment:
I'm not sure who the user of this is, but the format doesn't match Google JSON style that we use throughout the JSON output (properties use camelCase). It's an opportunity to fix that now.

Reply:
I've noticed the same when fixing this. One user is CBMC viewer, see #4863. We would just need to make sure we fix both of them in a synchronised way.